### PR TITLE
[risk=low] prod cdr version update

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
1. Updating prod cdr version with the new data generated from prod.

### CODE REVIEW

File: public-api/config/cdr_versions_prod.json [hunk 1]
```diff
@@ -2,13 +2,24 @@
...
+  {
+    "cdrVersionId": 2,
+    "isDefault": true,
+    "name": "All of Us Dataset v1",
+    "dataAccessLevel": 1,
+    "bigqueryProject": "aou-res-curation-prod",
+    "bigqueryDataset": "R2018Q4R1",
+    "creationTime": "2018-11-13 15:09:27Z",
+    "releaseNumber": 1,
+    "numParticipants": 116460,
```
Comment (calbach) [inline public-api/config/cdr_versions_prod.json:22]: Workbench has 116157 here - is there a reason for the difference?

**Review by calbach — APPROVED** [review]

File: public-api/config/cdr_versions_prod.json (hunk 1 above)
Comment (SrushtiGangireddy) [reply public-api/config/cdr_versions_prod.json:22]: Not sure, seemed to have been like this before too. 
https://console.cloud.google.com/bigquery?project=all-of-us-rw-prod&authuser=1&j=bq:US:bquxjob_6200136e_16959a89c38&page=queryresults

Comment (calbach) [reply public-api/config/cdr_versions_prod.json:22]: Thanks, I remember asking about this actually. More context here: https://pmi-engteam.slack.com/archives/C5TB0DP63/p1543272631085200

**Review by calbach — APPROVED** [review]

**Comment by SrushtiGangireddy** [comment]
ohh yeah, thanks.

**Pull request merged**